CMakeLists.txt

```diff
@@ -201,6 +201,7 @@ if(PHYLANX_WITH_TESTS_BENCHMARKS OR PHYLANX_WITH_TESTS_REGRESSIONS OR PHYLANX_WI
 endif()
 
 # Configure phylanxrun.py
+find_package(MPI QUIET)
```

Comment:
This should be executed only if HPX was configured with MPI itself (`HPX_WITH_PARCELPORT_MPI` is equal to `On`).